package.json

```diff
@@ -35,20 +36,26 @@
 		"vite": "^4.3.9",
 		"vite-plugin-environment": "^1.1.3"
 	},
-	"type": "module",
 	"dependencies": {
-		"@dfinity/agent": "^1.4.0",
-		"@dfinity/auth-client": "^1.3.0",
-		"@dfinity/candid": "^1.3.0",
+		"@dfinity/agent": "^2.1.2",
+		"@dfinity/auth-client": "^2.1.2",
+		"@dfinity/candid": "^2.1.2",
 		"@dfinity/gix-components": "^4.6.0",
-		"@dfinity/identity": "^1.3.0",
+		"@dfinity/identity": "^2.1.2",
 		"@dfinity/ledger-icp": "next",
 		"@dfinity/ledger-icrc": "^2.4.0",
-		"@dfinity/principal": "^1.3.0",
+		"@dfinity/principal": "^2.1.2",
 		"@dfinity/utils": "next",
 		"@esbuild-plugins/node-globals-polyfill": "^0.2.3",
 		"@esbuild-plugins/node-modules-polyfill": "^0.2.2",
+		"@nfid/identitykit": "^0.0.5",
 		"@rollup/plugin-inject": "^5.0.5",
+		"@slide-computer/signer": "^3.6.4",
+		"@slide-computer/signer-agent": "^3.6.4",
+		"@slide-computer/signer-transport-auth-client": "^3.6.4",
+		"@slide-computer/signer-transport-plug": "^3.6.4",
+		"@slide-computer/signer-web": "^3.6.4",
+		"@types/bignumber.js": "^4.0.3",
```

Comment:
Why did you introduce this? Do we really need this? How different is this from `bigint`?

Reply:
We have been using the `bignumber.js` package for a long time now. It's different because it handles exact operation on floats. It's more convenient to use than bigint to make calculations. But one is equivalent to the other. Switching to bigint would require a lot of refactoring.

Reply:
However they were two imports of 'bignumber.js'. Got rid of the unused one